### ISSUE TITLE
Contacts: Search function does not work

### ISSUE THREAD

**Issue by MartaLindstrom** [idmc-labs/idmc-database-feedback#300]
Cannot manage to use the keyword search to find contacts (e.g. by searching for their names)

**Comment by luisamg** [comment]
True. The search function works only with the first name and not with the surname. So, if I type the surname, nothing happens.

**Comment by MartaLindstrom** [comment]
This works now (which is great!), however only for names. Would it be possible to also create a search function for the country/organisation fields?
